tests/test_readwrite.py

```diff
@@ -753,7 +756,10 @@ def test_scanpy_pbmc68k(tmp_path, diskfmt, diskfmt2):
 @pytest.mark.skipif(not find_spec("scanpy"), reason="Scanpy is not installed")
 def test_scanpy_krumsiek11(tmp_path, diskfmt):
     filepth = tmp_path / f"test.{diskfmt}"
-    import scanpy as sc
+    with pytest.warns(
+        FutureWarning, match=r"Importing read_.* from `anndata` is deprecated"
+    ):
+        import scanpy as sc
```

Comment:
just importing scanpy shouldn’t trigger a warning.

Reply:
It imports `read_csv` etc. from top-level instead of `io` because the feature doesn't exist yet as far as `scanpy` is concerned

Reply:
Issue created with a to-do list

Reply:
Then these blocks should be changed to ignore the warnings instead of testing that they’re raised.

Otherwise these tests would start failing once things are fixed on the scanpy side.

Reply:
Ah, you ended up doing that here anyway: 1174453a4cc7fb5be135548f531b6cf895a3cae9

Reply:
Done!